source/includes/administration/_users.md

```diff
@@ -174,7 +209,13 @@ curl -X POST "https://cloudmc_endpoint/v2/users" \
 }
 ```
 
-Create a user in a specific organization. There's two different types of [role](#administration-roles) you can assign to the user. A system role will determine the set of system permissions the user will have. An environment role will give the user access to an environment and will determine what he can see and do in that environment. You will need the `Create a new user` permission to execute this operation.
+Create a user in a specific organization.
+
+Users are created asynchronously on the underlying service(s) and progress is reflected in the state of the user's service connection(s).
+
+There's two different types of [role](#administration-roles) you can assign to the user. A system role will determine the set of system permissions the user will have. An environment role will give the user access to an environment and will determine what he can see and do in that environment.
```

Comment:
there are two different types

Reply:
last sentence `what he can see` -> `what one can see`

Reply:
instead of `see and do` maybe use `access rights` or something else?

Reply:
I rewrote the sentence a bit. 493047c4621e951d804d9017c3f60d0b543f06d4